workflows/grzqc.nf

```diff
@@ -164,9 +184,38 @@ workflow GRZQC {
         }
         .set { samplesheet_ch }
 
-    // Run FASTQC on FASTQ files - per lane
+    samplesheet_ch.reads
+        .branch { meta, _reads ->
+            lng: meta.libraryType.endsWith('_lr')
+            srt: true
+        }
+        .set { samplesheet_ch_reads }
+
+    samplesheet_ch.alignments
+        .branch { meta, _alignment ->
+            lng: meta.libraryType.endsWith('_lr')
+            srt: true
+        }
+        .set { samplesheet_ch_alignments }
+
+    samplesheet_ch_reads.lng
+        .branch { meta, _reads ->
+            bam: meta.reads_filetype == "bam"
+            fastq: meta.reads_filetype == "fastq"
+        }
+        .set { samplesheet_ch_reads_lng }
+
+    // Index PacBio long-read BAMs
+    PBTK_PBINDEX(samplesheet_ch_reads_lng.bam)
+    ch_versions = ch_versions.mix(PBTK_PBINDEX.out.versions)
+
+    // Convert PacBio long-read BAMs to FASTQs
+    PBTK_BAM2FASTQ(samplesheet_ch_reads_lng.bam.join(PBTK_PBINDEX.out.pbi))
```

Comment:
Note to self: double-check if we need to index PacBio input BAMs or if we can pass `[]` because indexing takes a while and I can't see from the docs if an index is required or not.